public/src/models/banner.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ArticlesViewedSettings,
+  Cta,
+  Test,
+  UserCohort,
+} from '../components/channelManagement/helpers/shared';
+import { Region } from '../utils/models';
+
+export enum BannerTemplate {
```

Comment:
I had to move the banner models into a new file because a bug in webpack was causing import errors at runtime in specific places.
We should move all models into this directory, but I don't want this PR to get too big!